packages/nodes-base/nodes/Github/Github.node.ts

```diff
@@ -15,8 +15,8 @@ import {
 	githubApiRequestAllItems,
 	isBase64,
 } from './GenericFunctions';
-
-import { getRepositories, getUsers } from './SearchFunctions';
+import { getRepositories, getUsers, getWorkflows } from './SearchFunctions';
+import { validateJSON } from '../ClickUp/GenericFunctions';
```

Comment:
Is this needed if the same function has been added to the GenericFunctions?

Reply:
So basically if it was part of the generic functions i would import it from there

Reply:
You're right!

I will change the import path.